packages/server/graphql/mutations/refreshPokerEstimateIntegration.ts

```diff
@@ -0,0 +1,125 @@
+import {GraphQLID, GraphQLNonNull} from 'graphql'
+import {SubscriptionChannel} from 'parabol-client/types/constEnums'
+import {getUserId, isTeamMember} from '../../utils/authorization'
+import publish from '../../utils/publish'
+import type {GQLContext} from '../graphql'
+import RefreshPokerEstimateIntegrationPayload from '../types/RefreshPokerEstimateIntegrationPayload'
+
+const refreshPokerEstimateIntegration = {
+  type: new GraphQLNonNull(RefreshPokerEstimateIntegrationPayload),
+  description: 'Refresh the integration data for a poker estimate task',
+  args: {
+    taskId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The ID of the task to refresh'
+    },
+    meetingId: {
+      type: new GraphQLNonNull(GraphQLID),
+      description: 'The ID of the poker meeting'
+    }
+  },
+  resolve: async (
+    _source: unknown,
+    {taskId, meetingId}: {taskId: string; meetingId: string},
+    {authToken, dataLoader, socketId: mutatorId}: GQLContext
+  ) => {
+    const viewerId = getUserId(authToken)
+    const operationId = dataLoader.share()
+    const subOptions = {mutatorId, operationId}
+
+    // AUTH
+    const meeting = await dataLoader.get('newMeetings').load(meetingId)
+    if (!meeting) {
+      return {error: {message: 'Meeting not found'}}
+    }
+    if (meeting.meetingType !== 'poker') {
+      return {error: {message: 'Not a poker meeting'}}
+    }
+    const {endedAt, teamId} = meeting
+    if (!isTeamMember(authToken, teamId)) {
+      return {error: {message: 'Not on team'}}
+    }
+    if (endedAt) {
+      return {error: {message: 'Meeting has ended'}}
+    }
+
+    // RESOLUTION
+    const task = await dataLoader.get('tasks').load(taskId)
+    if (!task) {
+      return {error: {message: 'Task not found'}}
+    }
+    if (task.teamId !== teamId) {
+      return {error: {message: 'Task not on team'}}
+    }
+
+    const {integration} = task
+
+    // Clear relevant DataLoader caches based on integration type
+    if (integration) {
+      const {service} = integration
+
+      switch (service) {
+        case 'jira': {
+          const {cloudId, issueKey, accessUserId} = integration
+          // Clear the jiraIssue cache for this specific issue
+          dataLoader.get('jiraIssue').clear({
```

Comment:
this busts the dataloader cache (which is empty because this mutation never called it)